Description: Allow building with older factory.
 We only have version 4.1.1 in unstable.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2021-01-19

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -1358,7 +1358,7 @@
 if test $BUILD_factory = no
 then
     AC_MSG_CHECKING([whether factory library is installed])
-    FACTORY_MINVERSION=4.2.0
+    FACTORY_MINVERSION=4.0.0
     if $PKG_CONFIG --exists "factory >= $FACTORY_MINVERSION"
     then
 	FACTORY_NAME=factory
